apps/api/tests/api/application-controller.test.ts

```diff
@@ -227,24 +223,26 @@ describe('Application API', () => {
 
 	describe('Reject Application', () => {
 		it('should successfully reject an application in DAC_REVIEW state', async () => {
-			const applicationRecordsResult = await testApplicationRepo.listApplications({ user_id });
-			const rejectionReason = 'Reject';
+			const testApp = await getFirstApplicationTestByState(ApplicationStates.DAC_REVIEW);
 
-			assert.ok(applicationRecordsResult.success);
-			assert.ok(
-				Array.isArray(applicationRecordsResult.data.applications) && applicationRecordsResult.data.applications[0],
-			);
-
-			const { id } = applicationRecordsResult.data.applications[0];
-			await testApplicationRepo.findOneAndUpdate({ id, update: { state: ApplicationStates.DAC_REVIEW } });
+			const rejectionReason = 'Reject';
 
-			const result = await dacRejectApplication({ applicationId: id, rejectionReason });
+			const result = await dacRejectApplication({ applicationId: testApp.id, rejectionReason });
 			assert.ok(result.success);
 
-			const rejectedApplication = await getApplicationById({ applicationId: id });
+			const rejectedApplication = await getApplicationById({ applicationId: testApp.id });
 			assert.ok(rejectedApplication.success);
 			assert.strictEqual(rejectedApplication.data.state, ApplicationStates.REJECTED);
 		});
+
+		it('should failed to reject an application in NOT in DAC_REVIEW state', async () => {
```

Comment:
Someone is going to be confused by the grammar in this test message one day, could be improved ;)